### DISCUSSION TITLE
onthisday/03/03-22/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#14] [General]
# onthisday/03/03-22/

Details for On This Day are genarated by the posts added to the website so the content is subject to changes/updates over time.

https://fanzoflenazavaroni.github.io/onthisday/03/03-22/

**Comment by StephenMolloy1** [comment]
I remember that you saying that you hadn't read "Personality". Andrew O'Hagen's article in Granta magazine is on the internet archive, it includes a chapter from the book.
https://archive.org/details/granta7900ianj/mode/2up

**Reply by FanzOfLenaZavaroni** [reply]
Thanks. for thinking of me and sending the link.
